Include path in jekyll new commands (Usage docs) (#7931)
jekyll/jekyll: docs/_docs/usage.md

You can use this command in a number of ways:

> * `jekyll new .` - Creates a new Jekyll site in present working directory with default gem-based theme. The `.` can be replaced with a path. The path will be created if it does not exist.

Reviewer: Please recommend `jekyll new PATH` instead.
`jekyll new .` will throw an error if the current directory is not empty. In that situation, one will need to use `--force`:
```
jekyll new . --force
```

```suggestion
* `jekyll new some/path` - Creates a new Jekyll site with default gem-based theme at `./some/path`. The directories will be created as necessary.
```

> * `jekyll new --blank .` - Creates a new blank Jekyll site scaffold in the present working directory.

Reviewer: Similarly, here as well..
```suggestion
* `jekyll new some/path --blank` - Creates a new blank Jekyll site scaffold at path `./some/path`.
```

Reviewer 2: Let's make it consistent with the previous line (either `jekyll new [path] --blank` or `jekyll new PATH --blank`)

Reviewer: I had suggested `jekyll new some/path` in the previous line, so `jekyll new some/path --blank` here..
@MSVO If you choose to go with the placeholder suggestion, opt for `jekyll new PATH` and
`jekyll new PATH --blank` for consistency with output from command `jekyll help new`
